lib/src/components/NxTransferList/NxTransferList.tsx

```diff
@@ -9,13 +9,14 @@ import classnames from 'classnames';
 import { faPlusCircle, faTimesCircle } from '@fortawesome/free-solid-svg-icons';
 import { toLower, filter, includes, groupBy, partial, identity, prop, pipe } from 'ramda';
 
-import { Props, TransferListItemProps, DataItem, FilterFn } from './types';
+import DataItem from '../../util/DataItem';
+import { Props, TransferListItemProps, FilterFn } from './types';
 import NxFilterInput from '../NxFilterInput/NxFilterInput';
 import NxFontAwesomeIcon from '../NxFontAwesomeIcon/NxFontAwesomeIcon';
```

Comment:
Not sure if `NxTransferList` changes are supposed to be here.

Reply:
> Not sure if NxTransferList changes are supposed to be here.

Yes. This PR moves `DataItem` from `NxTransferList`'s types to a general utility file